services/ui-src/src/components/GridWithTotals/GridWithTotals.js

```diff
@@ -16,6 +16,14 @@ const GridWithTotals = props => {
     updateTotals();
   }, []); // eslint-disable-line react-hooks/exhaustive-deps
 
+  const roundValue = value => {
```

Comment:
I don't think this roundValue function should be contained in this component.  If it's a formatting function, and adds some functionality to this component that we can use somewhere else, it should be contained elsewhere and imported into the component for the purposes of use, but nothing about it ties it to this component specifically.

Reply:
additionally, I'm not sure formatting should be conditional based on whether or not the value contains decimal points.  it is either rounded or isn't.  Precision for all numerical data is usually the same at lest throughout the component

Reply:
I can definitely move it to a new file. The reasons I put it in this component was because, as far as I know, it is the only component that will be handling values with a decimal for this entire project. 

The reason the rounding is a conditional is because either we are talking about a whole number, which doesn't need to be rounded and also doesn't need to display a decimal, or we are talking about a calculated value, which will most likely result in a value with several decimal places and needs to be rounded.

Reply:
but rounding a whole number will not result in an error.  and it will get rid of the need to have a conditional.  

as far as decimal precision, I believe 1 digit is called for.  it is normally applied to all numbers being displayed unless explicitly called for otherwise.

I understand that for right now, as far as we know this rounding takes place in only this component, but logically this functionality is not tied to this component ONLY, and should not be included in it.  it is a formatting function on something this component provides

Reply:
I will remove this function from this component and move it to a proper location. 

The reason that the conditional is there is so that when we are handed a whole number, it doesn't try to round it and add an unnecessary decimal point. I think it looks better without every whole number being listed with a __.0. If you think that is is not needed then I can remove it.

Reply:
as previously stated, numbers are normally handled in a similar fashion regardless of their format, unless very explicitly called for otherwise.  it is not a matter of personal preference, but more an engineering standard.

if numerical output is rounded and displayed, it should be handled similarly for the entirety of said numerical output.  at least as far as the data set in question is concerned